### DISCUSSION TITLE
AstraDB & JanusGraph

### DISCUSSION THREAD

**Discussion by CashZarAdmin** [JanusGraph/janusgraph#4485] [Q&A] [answered]
I am trying to setup janusgraph to work with astradb, and I am following this, https://awesome-astra.github.io/docs/pages/tools/databases/janusgraph/, but it asks for secure connection zip file which is not available in the dashbaord and I thought this should be way simpler than this. Any help to connect JanusGraph to AstraDB? Thanks

**Comment by FlorianHockmann** [comment, marked as answer]
I've never used AstraDB, but the documentation that you have linked points to this document for _Secure Connect Bundle_: https://awesome-astra.github.io/docs/pages/astra/download-scb/

Doesn't that work for you?

You could also try to follow the AstraDB documentation in the JanusGraph docs: https://docs.janusgraph.org/storage-backend/cassandra/#deploying-on-datastax-astra
